src/client_lib/Makefile

```diff
@@ -66,7 +66,7 @@ lib/$(OUTPUT).a: $(OBJECTS)
 	
 lib/$(OUTPUT).so.$(VERSION): $(SHARED_OBJECTS)
 	$(GCC) -o $@ $(CFLAGS) $(SHARED_FLAGS) $(SHARED_OBJECTS) $(LIBS)
-	@ln -s $@ $(OUTPUT).so
+	chdir lib; ln -s $@ $(OUTPUT).so
```

Comment:
Probably want to trim the 'lib/' from the first argument here as well?

Reply:
'chdir' is also problematic on platforms like MSYS2. 'cd lib && ln -s <a> <b>' is more compatible.